modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -285,6 +305,8 @@ If Redpanda runs in a shared environment, where multiple applications run on the
 
 You can enable overprovisioning by either setting the CPU request to a fractional value or setting `overprovisioned` to `true`.
 
+NOTE: Setting `resources.requests.cpu` to a fractional value, such as 200m, enables Kubernetes to schedule Pods alongside other workloads efficiently, ensuring fair resource distribution. However, this may impact Redpanda's performance under heavy loads.
```

Comment:
This is where things start to get a bit nuanced. redpanda can't effectively make use of fractional CPU values. So the `--smp` flag is always rounded down with a minimum value of one. So 200m (--smp=1) will only work with the `overprovisioned` flag and `2500m` (--smp=2) will only result in redpanda utilizing 2 cores.

Reply:
is this true for `resources.cpu.cores`? Our docs currently say that fractional CPU values are allowed for that config.

I think given this:

> resources.{requests,limits} is mutually exclusive with resources.{memory,cpu,container}. If requests and limits are provided, memory, cpu, and container, will be ignored.


I should revert changes to the shared environments section since users will still need to use `resources.cpu` settings. Perhaps we need a note like:

```asciidoc
NOTE: When `resources.requests` or `resources.limits` are set, the `resources.cpu` parameter (including cores) is ignored. Ensure that you have not configured CPU requests and limits explicitly to avoid unexpected behavior in shared environments.
```